subliminal/providers/opensubtitles.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, language, hearing_impaired, page_link, subtitle_id, matched_b
         self.movie_imdb_id = movie_imdb_id
         self.series_season = series_season
         self.series_episode = series_episode
+        self.encoding = encoding
```

Comment:
`encoding` is a property of the parent `Subtitle` class, put it in the super call and edit the parent class so the constructor accepts the encoding argument.